runtime/interpreter/src/interpreter.cpp

```diff
@@ -440,14 +441,44 @@ int handleJumpLl(ShrimpVM *vm)
     auto rs_idx = instr.getRs();
     auto offset = instr.getJumpOffset();
 
-    auto ll = vm->acc().getValue() < frame.getReg(rs_idx).getValue();
+    auto ll = bit::getValue<int32_t>(vm->acc().getValue()) < bit::getValue<int32_t>(frame.getReg(rs_idx).getValue());
 
     std::cout << "LOG: " << instr.toString() << std::endl;
 
     vm->pc() += ll ? offset : instr.getByteSize();
     return dispatch_table[getOpcode(vm->pc())](vm);
 }
 
+int handleI32tof(ShrimpVM *vm)
+{
+    auto instr = Instr<InstrOpcode::I32TOF>(vm->pc());
+
+    auto acc = vm->acc().getValue();
+    auto acc_i = bit::getValue<int32_t>(acc);
```

Comment:
Maybe change in accordance with other handlers:
`int32_t acc_i32 = vm->acc().getValue();`

Reply:
Changed